api/CampusService.proto

```diff
@@ -160,3 +213,61 @@ message GetTopNewsReply {
   google.protobuf.Timestamp from = 4;
   google.protobuf.Timestamp to = 5;
 }
+
+
+message Cafeteria {
+  string cafeteria = 1;
```

Comment:
I wanted to use the names of the EAT-Api, since they are the somewhat official names. As far as I know, the EAT API integration should be finished in the android app in the next couple of weeks. Otherwise, I can add an intermediate solution in the android app which gets the names from the eat api.